### DISCUSSION TITLE
v1.0.0-rc.1

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/image#856] [Announcements]
> Work on updating the image module for Nuxt 3 began on [Jul 8, 2022](https://github.com/nuxt/image/discussions/548). After months of progress and continuous feedback, this is the first (semi-stable) version of the Image module ready for Nuxt 3 🚀 

## 🚦 Roadmap to v1

This is the first release candidate (rc1). We plan a quick RC process, and over that period, we plan to mainly stabilize the module and add some enhancements:

- Addressing most demanded feature requests and bug fixes
- Support multi-sources and server-side caching using Nuxt 3's built-in [Storage Layer](https://nuxt.com/docs/guide/directory-structure/server#server-storage) for the default image optimizer ([unjs/ipx](https://github.com/unjs/ipx))
- Better support for screen densities
- Better support for static output targets (we are tracking key issues)
- Integrate more closely with Nuxt 3 Core

## ✅ Upgrading

If you have been using the edge channel, you can now switch back to the main `@nuxt/image` package name. You can use the `rc` tag which will pull in the latest release candidate.

```diff
  {
    "devDependencies": {
-     "@nuxt/image-edge": "latest",
+     "@nuxt/image": "rc",
      "nuxt": "latest"
    }
  }
```

You should also update your `nuxt.config`:

```diff
  export default defineNuxtConfig({
    modules: [
-     '@nuxt/image-edge'
+     '@nuxt/image'
    ],
  })
```

<hr /><em>This discussion was created from the release <a href='https://github.com/nuxt/image/releases/tag/v1.0.0-rc.1'>v1.0.0-rc.1</a>.</em>

**Comment by mctweb** [comment]
Thank you! This was the final piece of the puzzle for us. Nuxt isn't just a framework. It's **the** framework.
### All Hail Nuxt

**Comment by DavidDeSloovere** [comment]
Counting down the hours... 😁
